artifacts/mysql/mysql_db_import.yml

```diff
@@ -1,4 +1,18 @@
 ---
-- tasks:
-  - name: Import the DB 
-    mysql_db: name={{db_name}} state=import target={{db_data}} login_user={{db_user}} login_password={{db_password}}
\ No newline at end of file
+- hosts: localhost
+  connection: local
+  tasks:
+  - name: Dump DB content in a temp file (base64 encoded)
```

Comment:
@MatMaul until we know if this recipe will work, I think that is better to maintain the old one as it is used in one of the TOSCA recipes. Please leave the original recipe so I can accept the pull request and start testing with the "host" and "connection" items.